### PR TITLE
BUGFIX: SD-334 hide stack labels when stackMeasuresToPercent is true

### PR DESCRIPTION
**Pull request by naibinh**
Changes:
- Check and disable stack labels in primary axis when `stackMeasuresToPercent` is `true`
- Update 4 storybooks: since stack labels in primary axis are disable, so it gets more space to show stack labels in secondary axis.
---
## PR Checklists
- [ ] Squash commits
- [x] Successful unit test
- [x] Successful storybook test
- [x] Successful AD testcafe
- [x] Successful KD testcafe
---
## Related PRs
AD: https://github.com/gooddata/gdc-analytical-designer/pull/2109
KD: https://github.com/gooddata/gdc-dashboards/pull/1761

### CODE REVIEW

**Comment by naibinh** [comment]
extended test - storybook

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/3659/ : SUCCESS in 11s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/2923/ : SUCCESS in 16m 05s

**Comment by yenkins** [comment]
Link to Storybook test result:

https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-storybook-zuul-docker/1568/artifact/test-storybook/html-report/index.html

**Comment by gdgate** [comment]
Build failed (check-extended pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/3660/ : SUCCESS in 28s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-pull-request-dispatcher/963/ : UNSTABLE in 14m 43s

**Comment by naibinh** [comment]
extended test - storybook

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/3664/ : SUCCESS in 15s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/2928/ : SUCCESS in 3m 21s

**Comment by yenkins** [comment]
Link to Storybook test result:

https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-storybook-zuul-docker/1570/artifact/test-storybook/html-report/index.html

**Comment by gdgate** [comment]
Build failed (check-extended pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/3665/ : SUCCESS in 10s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-pull-request-dispatcher/964/ : UNSTABLE in 24m 39s

**Comment by naibinh** [comment]
extended test - storybook

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/3680/ : SUCCESS in 23s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/2938/ : SUCCESS in 5m 59s

**Comment by yenkins** [comment]
Link to Storybook test result:

https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-storybook-zuul-docker/1574/artifact/test-storybook/html-report/index.html

**Comment by gdgate** [comment]
Build succeeded (check-extended pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/3681/ : SUCCESS in 20s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-pull-request-dispatcher/971/ : SUCCESS in 17m 57s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/3689/ : SUCCESS in 8s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/2944/ : SUCCESS in 7m 14s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/3691/ : SUCCESS in 6s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/2946/ : SUCCESS in 2m 17s

File: src/components/visualizations/chart/highcharts/plugins/autohideLabels/autohideColumnLabels.ts [hunk 1]
```diff
@@ -117,13 +117,13 @@ function getStackLabelPointsForDualAxis(stacks: any[]) {
     return flatten(
         // 'column0' is primary axis and 'column1' is secondary axis
         zip(...stacks.map((item: any, index: number) => values(item[`column${index}`])))
-    );
+    ).filter((point: any) => point);
```
Comment (imnutz) [inline src/components/visualizations/chart/highcharts/plugins/autohideLabels/autohideColumnLabels.ts]: ```Lodash.identity``` could help

File: src/components/visualizations/chart/highcharts/plugins/autohideLabels/autohideColumnLabels.ts [hunk 2]
```diff
@@ -117,13 +117,13 @@ function getStackLabelPointsForDualAxis(stacks: any[]) {
...
 }
 
 function toggleStackedLabelsForDualAxis() {
     const { yAxis } = this;
 
-    const stackTotalGroups = yAxis.map((axis: any) => axis.stackTotalGroup);
+    const stackTotalGroups = yAxis.map((axis: any) => axis.stackTotalGroup).filter((group: any) => group);
```
Comment (imnutz) [inline src/components/visualizations/chart/highcharts/plugins/autohideLabels/autohideColumnLabels.ts]: ```Lodash.identity``` could help

**Comment by gdgate** [comment]
Build failed (check-teardown pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-examples-stop-zuul-docker/428/ : FAILURE in 1m 24s

**Pull request closed without merging**